test/inductor/test_torchinductor_opinfo.py

```diff
@@ -349,8 +349,8 @@ def format_op(op):
     # a deconvolution forward propagation primitive
     "nn.functional.conv_transpose2d": {f32, f64},
     "nn.functional.conv_transpose3d": {f32, f64},
-    # rrelu not supported on XPU now
-    "nn.functional.rrelu": {f16, f32, f64},
+    # frexp not supported on XPU now
+    "frexp": {f16, f32, f64},
```

Comment:
Why do we remove `rrelu`? We have supported it?

Reply:
Yes